[2024-11-27T18:37:56.575+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-27T18:37:56.586+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.get_vars scheduled__2024-11-26T00:00:00+00:00 [queued]>
[2024-11-27T18:37:56.593+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.get_vars scheduled__2024-11-26T00:00:00+00:00 [queued]>
[2024-11-27T18:37:56.594+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-27T18:37:56.609+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_vars> on 2024-11-26 00:00:00+00:00
[2024-11-27T18:37:56.615+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-27T18:37:56.616+0000] {standard_task_runner.py:72} INFO - Started process 100 to run task
[2024-11-27T18:37:56.616+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'get_vars', 'scheduled__2024-11-26T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpzp188eku']
[2024-11-27T18:37:56.619+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask get_vars
[2024-11-27T18:37:56.658+0000] {task_command.py:467} INFO - Running <TaskInstance: ETL.get_vars scheduled__2024-11-26T00:00:00+00:00 [running]> on host b6d44e76794e
[2024-11-27T18:37:56.964+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='get_vars' AIRFLOW_CTX_EXECUTION_DATE='2024-11-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-26T00:00:00+00:00'
[2024-11-27T18:37:56.965+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-27T18:37:57.019+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-27T18:37:57.025+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-27T18:37:57.026+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=get_vars, run_id=scheduled__2024-11-26T00:00:00+00:00, execution_date=20241126T000000, start_date=20241127T183756, end_date=20241127T183757
[2024-11-27T18:37:57.072+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-27T18:37:57.253+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-27T18:37:57.257+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
